salesloft/read.go

```diff
@@ -43,5 +44,12 @@ func (c *Connector) buildReadURL(config common.ReadParams) (*urlbuilder.URL, err
 
 	link.WithQueryParam("per_page", strconv.Itoa(DefaultPageSize))
 
+	if !config.Since.IsZero() {
+		// documentation states ISO8601, while server accepts different formats
```

Comment:
can you link to the documentation page?

Reply:
Updated comment, see:
https://github.com/amp-labs/connectors/blob/2a37a22d11404e3eefe95f591738d89dd07f0c98/salesloft/read.go#L48-L53